### ISSUE TITLE
lr_scheduler

### ISSUE THREAD

**Issue by shiningsunnyday** [buzkent86/WikiSatNet#1]
lr_scheduler isn't defined in im2text_pretraining.py

**Comment by uzkent** [comment]
Sorry, for the late reply. You do not need to schedule learning rate. I pushed my update.
